src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -476,5 +476,25 @@ private async Task<PackageSpec> GetPackageSpecAsync(ISettings settings)
                 }
             };
         }
+
+        internal override async Task<IList<LockFileTarget>> GetFullRestoreGraphAsync(CancellationToken token)
```

Comment:
This can go in the PackageReferenceProject that both classesimplement: https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/PackageReferenceProject.cs

Reply:
Fixed.